Sources/SourceKitLSP/LanguageService.swift

```diff
@@ -146,7 +146,12 @@ public protocol LanguageService: AnyObject, Sendable {
   func completion(_ req: CompletionRequest) async throws -> CompletionList
   func hover(_ req: HoverRequest) async throws -> HoverResponse?
   func symbolInfo(_ request: SymbolInfoRequest) async throws -> [SymbolDetails]
-  func openGeneratedInterface(_ request: OpenGeneratedInterfaceRequest) async throws -> GeneratedInterfaceDetails?
+  func openGeneratedInterface(
+    textDocument: TextDocumentIdentifier,
```

Comment:
`TextDocumentIdentifier` is just a wrapper around `DocumentURI` and `DocumentURI` is generally easier to work with, so I’d just use `DocumentURI` here.

```suggestion
    document: DocumentURI,
```

Reply:
Thank you @ahoppen for your review! I've committed the recommended changes accordingly.